assets/src/edit-story/components/library/panes/text/textSets/textSetsPane.js

```diff
@@ -154,7 +156,11 @@ function TextSetsPane({ paneRef }) {
           {PANE_TEXT.TITLE}
         </Headline>
         <TextSetsToggle>
-          <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}>
+          <Text
+            as="label"
+            htmlFor={toggleId}
+            size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+          >
```

Comment:
Realized the toggle in the text sets pane didn't have a `label` associated with the toggle. This can be moved to another pr if we'd like as well. There are no visual changes here other than making the cursor a `pointer` when hovering the text.